demos/defender/defender_demo_json/defender_demo.c

```diff
@@ -410,6 +416,32 @@ static bool collectDeviceMetrics( void )
         }
     }
 
+    /* Collect CPU usage time metrics from the system.
+     * This is an example of a custom metric of number-list type. */
+    if( metricsCollectorStatus == MetricsCollectorSuccess )
+    {
+        metricsCollectorStatus = GetCpuUsageStats( &customMetrics.cpuUsageStats );
+
+        if( metricsCollectorStatus != MetricsCollectorSuccess )
+        {
+            LogError( ( "GetCpuUsageStats failed. Status: %d.",
+                        metricsCollectorStatus ) );
+        }
+    }
+
+    /* Collect data on memory usage and calculate statistic on % of available memory
```

Comment:
Is it available memory or % of available memory?

Reply:
This is a stale comment, have updated it.